contrib/custom_top_level_templates.go

```diff
@@ -0,0 +1,103 @@
+package contrib
+
+import (
+	"github.com/solo-io/skv2/codegen/model"
+	"github.com/solo-io/skv2/contrib/codegen/funcs"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// use to get all cross-group templates in contrib
+func AllCrossGroupTemplates(params CrossGroupTemplateParameters) []model.CustomTemplates {
```

Comment:
As discussed on slack, this won't work as the `OutputFilename` will match for all 3 and overwrite